src/components/TransactionBlock/PlrDaoStakingTransactionBlock.tsx

```diff
@@ -0,0 +1,632 @@
+import { ethers } from 'ethers';
+import debounce from 'debounce-promise';
+
+// Types
+import { IPlrDaoStakingMembershipBlock } from '../../types/transactionBlock';
+
+import TextInput from '../TextInput';
+import SelectInput, { SelectOption } from '../SelectInput/SelectInput';
+import { useEtherspot, useTransactionBuilder } from '../../hooks';
+import {
+  formatAmountDisplay,
+  formatMaxAmount,
+} from '../../utils/common';
+import {
+  addressesEqual,
+  isValidEthereumAddress,
+  isValidAmount,
+} from '../../utils/validation';
+import AccountSwitchInput from '../AccountSwitchInput';
+import NetworkAssetSelectInput from '../NetworkAssetSelectInput';
+import {
+  Chain,
+  supportedChains,
+  plrDaoAsset,
+  plrDaoMemberNFT,
+} from '../../utils/chain';
+import { swapServiceIdToDetails } from '../../utils/swap';
+import { IAssetWithBalance } from '../../providers/EtherspotContextProvider';
+import { RoundedImage } from '../Image';
+import { Theme } from '../../utils/theme';
+import { DestinationWalletEnum } from '../../enums/wallet.enum';
+import Text from '../Text/Text';
+export interface IPlrDaoTransactionBlockValues {
+  hasEnoughPLR: boolean;
+  fromChainId?: number;
+  fromAssetAddress?: string;
+  fromAssetIconUrl?: string;
+  fromAsset?: IAssetWithBalance;
+  selectedAsset?: IAssetWithBalance | null;
+  toAsset?: TokenListToken;
+  fromAssetDecimals?: number;
+  fromAssetSymbol?: string;
+  amount?: string;
+  accountType: AccountTypes;
+  receiverAddress?: string;
+  offer?: ExchangeOffer;
+}
+
+const Title = styled.h3`
+  margin: 0 0 18px;
+  padding: 0;
+  font-size: 16px;
+  color: ${({ theme }) => theme.color.text.cardTitle};
+  font-family: 'PTRootUIWebBold', sans-serif;
+`;
+
+const WalletReceiveWrapper = styled.div`
+  display: flex;
+  flex-direction: row;
+`;
+
+const OfferDetails = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  font-family: 'PTRootUIWebMedium', sans-serif;
+`;
+
+const Container = styled.div`
+  align-items: center;
+  font-family: 'PTRootUIWebMedium', sans-serif;
+  background: #21002e;
+  color: #fefefe;
+  padding: 16px;
+  margin: 5px;
+  border-image: linear-gradient(#346ecd, #cd34a2) 30;
+  border-width: 2px;
+  border-style: solid;
+`;
+
+const Value = styled.div`
+  font-family: 'PTRootUIWebMedium', sans-serif;
+  color: #57c2d6;
+  display: contents;
+`;
+
+const Total = styled.div`
+  font-family: 'PTRootUIWebMedium', sans-serif;
+  color: #ff0065;
+  display: contents;
+  font-weight: bold;
+`;
+
+const HorizontalLine = styled.div`
+  margin: 9px 0;
+  width: 100%;
+  height: 2px;
+  background: linear-gradient(90deg, #23a9c9, #cd34a2);
+`;
+
+const Bold = styled.p`
+  font-weight: bold;
+`;
+
+//TODO : change color of block if polygon has less than 10,000 plr on keybased / smart wallet
+const Block = styled.div`
+  font-size: 12px;
+  padding: 2px;
+  text-indent: 2px;
+  display: flex;
+  color: ${(props) => props.color === 'red' && '#ff0065'};
```

Comment:
red is valid color name in css, make sure either to rename prop, just `red` prop name maybe? not sure what's the context, but this is wrong